deployments/helm/dra-example-driver/values.yaml

```diff
@@ -53,6 +53,8 @@ kubeletPlugin:
   nodeSelector: {}
   tolerations: []
   affinity: {}
+  kubeletRegistrarDirectoryPath: /var/lib/kubelet/plugins_registry
+  kubeletPluginsDirectoryPath: /var/lib/kubelet/plugins
```

Comment:
This is the only place where a default is stored and Helm is the only supported installation method, right? Otherwise we would need similar defaults elsewhere.

Do we have documentation of values that can be set?

Reply:
Right. I was wondering whether the helper package should have those constants... and it already does!

/lgtm
/approve